### ISSUE TITLE
Unsatisfiable classes due to 'prominent'

### ISSUE THREAD

**Issue by leechuck** [pato-ontology/pato#265]
The class PATO:0001482 (prominent) is a subclass of both prominence (a process quality) and protruding (an object quality), leading to unsatisfiable classes in ontologies that assert them to be disjoint. Please remove one of the superclasses.

**Comment by cmungall** [comment]
wow, that is kind of bonkers.

I was wondering who put "prominence" in there it's kind of terrible...

```
id: PATO:0015007
name: prominence
def: "A quality inhering in a bearer by virtue of the degree to which the bearers stands out or projects beyond a surface or line." [GOC:CJM]
subset: attribute_slim
is_a: PATO:0000057 ! occurrence
```

will fix

**Comment by cmungall** [comment]
@matentzn surprised we haven't seen this?

**Comment by cmungall** [comment]
ugh lots more

![image](https://user-images.githubusercontent.com/50745/64402665-a73b1780-d0b0-11e9-8858-3d59a3f93413.png)

![image](https://user-images.githubusercontent.com/50745/64402748-e79a9580-d0b0-11e9-887f-50e6136baaab.png)

I can't believe we have a class 'variant' in pato...

Anyway I have fixed the reported issue but there is a lot to do here

 - the whole prominence vs protruding thing needs work
 - work needed to resolve incoherencies (we are now failing on master, sorry...)
 - my Makefile commit fix adds more tests but we need to refactor things to work the ODK way

Have to work on something else now...

**Comment by matentzn** [comment]
@dosumis is usually handling PATO, I have not so far been involved in its development. 2 observations:

1. All of these unsatisfiable classes have been there for years, 5 as far as I can see;
2. Except for the case of 'prominent' that you have fixed, they are all a result of the fact that they (the unsatisfiable classes) are children of "object" and "process" qualities, which are disjoint in this pato_ext.owl. So, @dosumis if you want to fix this, you can go to these classes: 
![image](https://user-images.githubusercontent.com/7070631/64403814-ded69f00-d071-11e9-9011-d26696f05098.png)
And decide which of the two they (object or process qualities) they should be. After that we can run the release again.
3. ODK all by itself would not have caught this; pato as it is is coherent; its just this pato ext file which is not.. Probably would be better to have the external dependencies integrated in the usual way, and then release PATO as a simple artefact?

**Comment by matentzn** [comment]
As to the variant class, I will make a new ticket.

**Comment by cmungall** [comment]
> ODK all by itself would not have caught this; pato as it is is coherent; its just this pato ext file which is not

Yep, I added an ad-hoc test for pato_ext for now

But what I meant was that if we did this the ODK way then pato-edit would have the content of ext, we could still keep pato.owl as being basic so as not to mess people up. They thing is to have full axioms in view of editors.

I think we should just remove the disjointness axiom til we can release a coherent pato_ext

**Comment by matentzn** [comment]
Yeah, understood and agreed with this. Lets wait for David to chime in and it will be fixed in the (european) afternoon.

**Comment by dosumis** [comment]
Slightly shocked that the build didn't catch this.  Looking into it now.

**Comment by dosumis** [comment]
OK.  Still quite confused about how the build reached its current state. High properly move this to ODK rather than tinkering with Makefile. For now I'll
1.  Add a full set of import statements to editors file (it currently has none!) 
2. Remove the offending disjointness statement from pato_ext.owl
3. Check test release files - do we need to make simplified versions for release?
3. Make a ticket listing terms that need fixing before we can add the disjointness statment back.

@matentzn - Can you schedule work to move this to ODK?

**Comment by dosumis** [comment]
Fix for remaining inconsistencies here: https://github.com/pato-ontology/pato/pull/268
